.eslintrc.json

```diff
@@ -4,7 +4,7 @@
   },
   "plugins": ["jsdoc"],
   "extends": ["eslint:recommended", "prettier"],
-  "globals": {},
+  "globals": { "Set": true },
```

Comment:
`Set` has been supported since node `0.12`. The minimum node version has been bumped to support this.